src/pages/actions/[slug].tsx

```diff
@@ -120,6 +121,7 @@ export default function ActionPage() {
   const activeGoal = useReactiveVar(activeGoalVar);
   const site = useSite();
   const [activeSubAction, setActiveSubAction] = useState(undefined);
+  const theme = useContext(ThemeContext);
```

Comment:
You can use `useTheme` here, it's a bit shorter and the name has a clearer intention. [Here's](https://github.com/kausaltech/kausal-paths-ui/blob/main/src/components/Layout.tsx#L58) an example.

Reply:
Done, thanks!